### ISSUE TITLE
Allow Persistent Toolbar for Markdown Editor

### ISSUE THREAD

**Issue by ScriptInfra** [Sinacs/Joplin.Plugin.ReMoods.Theme#51]
Hello! I would like to submit a request to add a setting to disable the toolbar auto-hide for the markdown editor (shown below).

<img width="863" alt="Screenshot 2023-06-05 at 12 49 13 PM" src="https://github.com/Sinacs/Joplin.Plugin.ReMoods.Theme/assets/63025323/2d81810b-ee9e-40dd-82a6-369fe84d65f0">

I also noticed that WYSIWYG editor is configured to auto-hide the information in the note title (pictured below). Would it be possible to also create a setting to disable this auto-hide? The goal here being to set both the markdown editor and WYSIWYG editor to either auto-hide or not, for consistency purposes.

<img width="868" alt="Screenshot 2023-06-05 at 12 59 50 PM" src="https://github.com/Sinacs/Joplin.Plugin.ReMoods.Theme/assets/63025323/cda71c72-f0ec-47a6-8373-e0946c138d59">

**Comment by Sinacs** [comment]
Yes, it is possible for the "small toolbar" which at the top right, but currently not considering to implement this feature for the "main editor toolbar".

**Comment by ScriptInfra** [comment]
Thank you! Just wondering, is there a reason disabling the auto-hide is not advisable in the markdown editor?

**Comment by ScriptInfra** [comment]
Thank you! Looking forward to testing it!

**Comment by ScriptInfra** [comment]
Looks great, thank you so much!
